docs/en-US/component/progress.md

```diff
@@ -90,10 +90,10 @@ progress/striped-progress
 | status         | the current status of progress bar                                                    | ^[enum]`'success' \| 'exception' \| 'warning'` | —       |
 | indeterminate  | set indeterminate progress                                                            | ^[boolean]                                     | false   |
 | duration       | control the animation duration of indeterminate progress or striped flow progress     | ^[number]                                      | 3       |
-| color          | background color of progress bar. Overrides `status` prop                             | ^[enum]`'string' \| 'function' \| 'array'`     | ''      |
+| color          | background color of progress bar. Overrides `status` prop                             | ^[string] / ^[function] / ^[array]             | ''      |
 | width          | the canvas width of circle progress bar                                               | ^[number]                                      | 126     |
 | show-text      | whether to show percentage                                                            | ^[boolean]                                     | true    |
-| stroke-linecap | circle/dashboard type shape at the end path                                           | ^[enum]`'butt' \| 'round' \| 'square'`         | round   |
+| stroke-linecap | butt/circle/dashboard type shape at the end path                                      | ^[enum]`'butt' \| 'round' \| 'square'`         | round   |
 | format         | custom text format                                                                    | ^[Function]`(percentage: number) => string`    | —       |
 | striped        | stripe over the progress bar's color                                                  | ^[boolean]                                     | false   |
```

Comment:
Done. Confused with the later acceptable values 🤣

Reply:
Do you mean `striped`? Why？

Reply:
striped default value is `—`

Reply:
According to the [document](https://vuejs.org/guide/components/props.html#prop-validation#:~:text=The%20Boolean%20absent%20props%20will%20be%20cast%20to%20false.), if the boolen type is not passed, it will be converted to false by default. Maybe it is right to think that the default value is false?

Reply:
You are right, it appears that some of my previous defaults were wrong.